src/utilities/theme/color-adjustments.ts

```diff
@@ -12,6 +12,8 @@ export enum UNSTABLE_Color {
   Warning = '#FFC453',
   Highlight = '#59D0C2',
   Success = '#008060',
+  Peach = '#FFC8B5',
+  Cranberry = '#FDC7CE',
```

Comment:
What do you think about providing one avatar color and then changing the lightness and hue to generate the palette?

Reply:
I like the idea of providing a single decorative color, then perhaps adjusting based on the same hue delta as our colors to create the contrast/counter/secondary decorative color. In theory, regardless of what color is provided the resulted generated color should look nice.

That means these color roles would be relative adjustments instead of absolute ones, which is doable would just need some refactoring of the factory and new schema in the adjustments to flag this role as a relative adjustment.

Would like to talk through this a bit more though.